Specs/DateTime/Japanese/DateTimePeriodParser.json

```diff
@@ -220,14 +212,13 @@
     "Context": {
       "ReferenceDateTime": "2016-11-07T16:12:00"
     },
-    "NotSupported": "dotnet",
     "NotSupportedByDesign": "javascript,python,java",
     "Results": [
       {
         "Text": "今日の午後4時から明日の午後5時まで",
         "Type": "datetimerange",
         "Value": {
-          "Timex": "(2016-11-07T16,2016-11-08T17,PT25H)",
+          "Timex": "(2016-11-07T16:00:00,2016-11-08T17:00:00,PT25H)",
```

Comment:
This behaviour of timexes with or without unnecessary 00 terms is something we really should make consistent. Can you create an issue for it? Low priority for now though.

Reply:
ok will create